src/TSID/tests/QPFixedBaseTSIDTest.cpp

```diff
@@ -0,0 +1,303 @@
+/**
+ * @file QPFixedBaseTSIDTest.cpp
+ * @authors Ines Sorrentino
+ * @copyright 2021 Istituto Italiano di Tecnologia (IIT). This software may be modified and
+ * distributed under the terms of the GNU Lesser General Public License v2.1 or any later version.
+ */
+
+// Catch2
+#include <catch2/catch.hpp>
+
+// std
+#include <random>
+
+// BipedalLocomotion
+#include <BipedalLocomotion/Conversions/ManifConversions.h>
+#include <BipedalLocomotion/ParametersHandler/StdImplementation.h>
+#include <BipedalLocomotion/System/VariablesHandler.h>
+
+#include <BipedalLocomotion/TSID/JointTrackingTask.h>
+#include <BipedalLocomotion/TSID/SE3Task.h>
+
+#include <BipedalLocomotion/TSID/QPFixedBaseTSID.h>
+
+#include <BipedalLocomotion/ContinuousDynamicalSystem/FixedBaseDynamics.h>
+#include <BipedalLocomotion/ContinuousDynamicalSystem/ForwardEuler.h>
+
+#include <iDynTree/Core/EigenHelpers.h>
+#include <iDynTree/Model/ModelTestUtils.h>
+
+using namespace BipedalLocomotion::ParametersHandler;
+using namespace BipedalLocomotion::System;
+using namespace BipedalLocomotion::ContinuousDynamicalSystem;
+using namespace BipedalLocomotion::Conversions;
+using namespace BipedalLocomotion::TSID;
+
+constexpr auto robotAcceleration = "robotAccelration";
+constexpr auto jointTorques = "jointTorques";
+constexpr int maxNumOfContacts = 1;
+constexpr double dT = 0.01;
+
+struct TSIDAndTasks
+{
+    std::shared_ptr<TaskSpaceInverseDynamics> tsid;
+    std::shared_ptr<SE3Task> se3Task;
+    std::shared_ptr<JointTrackingTask> regularizationTask;
+};
+
+struct DesiredSetPoints
+{
+    manif::SE3d endEffectorPose;
+    Eigen::VectorXd joints;
+};
+
+struct System
+{
+    std::shared_ptr<ForwardEuler<FixedBaseDynamics>> integrator;
+    std::shared_ptr<FixedBaseDynamics> dynamics;
+};
+
+std::shared_ptr<IParametersHandler> createParameterHandler()
+{
+    auto parameterHandler = std::make_shared<StdImplementation>();
+    parameterHandler->setParameter("robot_acceleration_variable_name", robotAcceleration);
+    parameterHandler->setParameter("joint_torques_variable_name", jointTorques);
+    parameterHandler->setParameter("max_number_of_contacts", maxNumOfContacts);
+
+    parameterHandler->setParameter("verbosity", false);
+
+    /////// SE3 Task
+    constexpr double kp_se3task = 16.0;
+    constexpr double kd_se3task = 8.0;
+    auto SE3ParameterHandler = std::make_shared<StdImplementation>();
+
+    SE3ParameterHandler->setParameter("robot_acceleration_variable_name",
+                                      robotAcceleration);
+    SE3ParameterHandler->setParameter("kp_linear", kp_se3task);
+    SE3ParameterHandler->setParameter("kd_linear", kd_se3task);
+    SE3ParameterHandler->setParameter("kp_angular", kp_se3task);
+    SE3ParameterHandler->setParameter("kd_angular", kd_se3task);
```

Comment:
how does the hard-coded 0s in `createBaseTask()` in `FixedBaesQPTSID` affect these settings?